.github/workflows/release-nightly.yaml

```
@@ -161,7 +164,7 @@ jobs:
       - name: Update versions
         shell: bash
         run: |
-          version=0.0.0-nightly-$(date +"%y%m%d")
+          version=0.0.0-nightly

```

```suggestion
          version=0.0.0-nightly-$(date +"%y%m%d")
```
This line needs to be reverted as you suggested in a reply to @lpil's comment